### ISSUE TITLE
Add filters for price and stock

### ISSUE THREAD

**Issue by noonien** [yaqwsx/jlcparts#90]
There are times when the search yields a lot of results, it would be very helpful to be able to filter them by price and stock, to help make choosing easier.

**Comment by Luxtech** [comment]
Yes it would be awesome if you can specify a price range, to filter out expensive and or cheap parts (can parts be too cheap... No! :) )

Currently you can sort by price but this makes it impossible to sort by another category.
